### PR TITLE
Various website improvements

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@docusaurus/core": "3.5.2",
     "@docusaurus/preset-classic": "3.5.2",
+    "@iconify/react": "^5.0.2",
```
Comment (sschuberth) [inline website/package.json:20]: For the front-end UI we're using https://lucide.dev/icons/. Should we align?

Comment (mnonnenmacher) [reply website/package.json:20]: Lucide doesn't have the tool icons I need, for example, for Kubernetes.

File: ui/public/favicon.svg
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply ui/public/favicon.svg:1]: I guess so, I'm not the designer 😉

File: website/src/components/HomepageFeatures/index.tsx [hunk 2]
```diff
@@ -1,52 +1,102 @@
 import clsx from 'clsx';
 import Heading from '@theme/Heading';
 import styles from './styles.module.css';
+import {Icon} from '@iconify/react';
```
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply website/src/components/HomepageFeatures/index.tsx:4]: Yes, I think the whole formatting is currently off, that should be fixed in a dedicated PR. For now I just used the default formatting of IntelliJ.

File: website/src/components/HomepageFeatures/index.tsx [hunk 3]
```diff
@@ -1,52 +1,102 @@
...
 
 type FeatureItem = {
   title: string;
-  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  Svg?: React.ComponentType<React.ComponentProps<'svg'>>;
+  icon?: string;
```
[the remark this answers is not in the document]
Comment (mnonnenmacher) [reply website/src/components/HomepageFeatures/index.tsx:9]: Yes, but I expect that this is only a temporary solution anyway. I hope we can replace the homepage with a more sophisticated design sometime.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**